aadwarf64/aadwarf64.rst

```diff
@@ -470,11 +470,19 @@ integers.
    .. _Note 8:
 
    8. The RA_SIGN_STATE pseudo-register records whether the return address has
-      been signed with a PAC. This information can be used when unwinding. It
-      is an unsigned integer with the same size as a general register. Only
-      bit[0] is meaningful and is initialized to zero. A value of 0 indicates
-      the return address has not been signed. A value of 1 indicates the return
-      address has been signed.
+      been signed with a PAC, and whether the value of PC has been used as a
+      diversifier for the return address signing. This information can be used
+      when unwinding. It is an unsigned integer with the same size as a general
+      register. Only bit[0] and bit[1] are meaningful and are initialized to zero.
```

Comment:
I'd add a truth table somewhere maybe easier to follow.
Also would highlight the 0b10 setup is invalid, as indicate a incorrect step or codegen issue, the unwinder may want to stop in this case.

| bit 1 | bit 0 | state |
| -------- | ------- | ------- |
| 0 | 0 | LR is not signed |
| 0 | 1 | LR is signed with SP |
| 1 | 1 | LR is signed with SP+PC |
| 1 | 0 | Invalid state |